fonts.scss

```diff
@@ -1,7 +1,12 @@
-@import "@fontsource/open-sans/300";
-@import "@fontsource/open-sans/400-italic";
-@import "@fontsource/open-sans/400";
-@import "@fontsource/open-sans/600";
+@import "@fontsource/inter/100";
+@import "@fontsource/inter/200";
+@import "@fontsource/inter/300";
+@import "@fontsource/inter/400";
+@import "@fontsource/inter/500";
+@import "@fontsource/inter/600";
+@import "@fontsource/inter/700";
+@import "@fontsource/inter/800";
+@import "@fontsource/inter/900";
```

Comment:
Do we need all font weights? Will that load all the woff files of all sizes? As far as I saw 400, 500, 600, 700, 800 are used upper in the file.. more somewhere else or potentially by design team randomly?

Reply:
will switch to the variable font in the next commits so no explicit font weight files then